webviz_ert/models/response.py

```diff
@@ -27,10 +27,7 @@ def __init__(
         self._summary_misfits_df: Optional[pd.DataFrame] = None
         self._ensemble_size: int = ensemble_size
         self._active_realizations: List[int] = active_realizations
-        self._has_observations: Optional[bool] = None
-
-        if "has_observations" in resp_schema:
-            self._has_observations = resp_schema["has_observations"]
+        self._has_observations: Optional[bool] = resp_schema.get("has_observations")
```

Comment:
this need to be optional anymore?

Reply:
not anymore after your ert-storage change